plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -51,6 +99,335 @@ def get_ilo_sessions(self):
+                logical_drives_count = logical_drives_count + len(logical_drives_details)
+                result["logical_drives"] = logical_drives
+                result["logical_drives_count"] = logical_drives_count
+                # result["ret"] = True
+                return {
+                    "msg": result,
+                    "ret": True
+                }
+            # return result
+
+        result["logical_drives_details"] = logical_drives_details
+        # result["ret"] = True
+        return {
+            "msg": result,
+            "ret": True
+        }
+        # return result
+
+    def get_snmpv3_users(self):
+        # This method returns list of SNMPv3 users
+        snmpv3_users = []
+
+        response = self.get_request(self.root_uri + self.manager_uri)
+        if not response["ret"]:
+            return response
+
+        response = self.get_request(self.root_uri + self.manager_uri + "SnmpService/SNMPUsers/")
+        if not response["ret"]:
+            return response
+
+        snmp_list = response["data"]
+
+        for item in snmp_list["Members"]:
+            item_response = self.get_request(
+                self.root_uri + item["@odata.id"]
+            )
+            if not item_response["ret"]:
+                return item_response
+
+            output = self.remove_odata(item_response["data"])
+            snmpv3_users.append(output)
+
+        return {
+            "ret": True,
+            "msg": snmpv3_users
+        }
+
+    def get_snmp_alert_destinations(self):
+        # This method returns list of SNMP alert destinations
+        alert_destinations = []
+
+        response = self.get_request(self.root_uri + self.manager_uri)
+        if not response["ret"]:
+            return response
+
+        response = self.get_request(self.root_uri + self.manager_uri + "SnmpService/SNMPAlertDestinations/")
+        if not response["ret"]:
+            return response
+
+        snmp_list = response["data"]
+        for item in snmp_list["Members"]:
+            item_response = self.get_request(
+                self.root_uri + item["@odata.id"]
+            )
+            if not item_response["ret"]:
+                return item_response
+
+            # Remove odata details
+            output = self.remove_odata(item_response["data"])
+
+            # Get SNMPv3 users details attached to alert destination
+            if "SNMPv3User" in output:
+                response = self.get_request(
+                    self.root_uri + output["SNMPv3User"]["@odata.id"]
+                )
+                if not response["ret"]:
+                    return response
+                output["SNMPv3User"] = self.remove_odata(response["data"])
+            alert_destinations.append(output)
+
+        return {
+            "ret": True,
+            "msg": alert_destinations
+        }
+
+    def verify_bios_attributes(self, bios_attributes):
```

Comment:
Could this be moved to the standard Redfish module? There's nothing here that looks iLO-specific to me.

Reply:
Hi, 
Yes you're right. Removing it from here and will raise a separate PR to merge this to redfish_command module. Thanks!

Reply:
Resolving this conversation. Thanks!